catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/TestWfsFilterDelegate.java

```diff
@@ -138,12 +144,44 @@
 
   private String mockFeatureType = "mockFeatureType";
 
-  private String mockFeatureProperty = "mockFeatureProperty";
+  private String mockFeatureProperty = "localpart.EXACT_COLLECT_DATE";
 
   private MetacardMapper mockMapper;
 
   private FeatureMetacardType mockFeatureMetacardType = mock(FeatureMetacardType.class);
 
+  private List<Entry> mappingEntries;
+
+  @Before
+  public void setup() {
```

Comment:
This setup logic seems to repeated for WFS 1.0, WFS 1.1 and WFS 2.0. Could it be refactored into a common class?